li-utils/src/main/pegasus/com/linkedin/common/JoinUrn.pdl

```diff
@@ -0,0 +1,24 @@
+namespace com.linkedin.common
+
+/**
+ * Standardized join identifier.
+ */
+@java.class = "com.linkedin.common.urn.JoinUrn"
```

Comment:
This doesn't seem necessary to me. Can you explain the rationale here?

Reply:
Please see above comment.